python/python/bystro/supervised_ppca/gf_dropout_pt.py

```diff
@@ -31,11 +31,13 @@
 None
 """
 import numpy as np
+from numpy.typing import NDArray
+from typing import Dict, List, Optional
 
 from tqdm import trange
-import torch # type: ignore
-from torch import nn # type: ignore
-from torch.distributions.multivariate_normal import MultivariateNormal # type: ignore
+import torch  # type: ignore
```

Comment:
let's get rid of the # type: ignore here and below, none of these need type : ignore